src/worker/vdb_upload_worker.py

```diff
@@ -107,6 +112,45 @@ def generate_uuid_from_tuple(t, namespace_uuid='6ba7b810-9dad-11d1-80b4-00c04fd4
 
     return str(unique_uuid)
 
+def create_redis_source_chunk_dict(text_embeddings_list, batch_id, job_id):
```

Comment:
we are also storing `source_document` now, which is the filename, as of this PR [here](https://github.com/dgarnitz/vectorflow/pull/58) that just merged yesterday.

Reply:
Okay noted